python/python/tests/test_transaction.py

```diff
@@ -143,6 +144,11 @@ async def transaction_test(
     transaction.lpop_count(key5, 2)
     args.append([value2, value])
 
+    transaction.lpush(key10, [value, value2])
```

Comment:
why don't you re-use key5 instead of creating a new list...?

Reply:
key5 is an empty list at this point, so wasn't sure if we wanted to use it to test linsert. But yeah I can update to do a linsert on the empty list (which ends up being a no-op).